[2024-04-15T18:54:10.958-0400] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-15T18:54:10.980-0400] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: preprocessing_dag.bash_task scheduled__2024-01-16T00:00:00+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2024-04-15T18:54:10.983-0400] {local_task_job_runner.py:163} INFO - Task is not able to be run
